tests/integration/test_leases.py

```diff
@@ -157,7 +157,7 @@ def test_updatedDate_is_left_unchanged_when_given_invalid_params(self, auth_head
         with freeze_time(Time.one_year_from_now()):
             response = self.client.put(f'{self.endpoint}/{self.lease.id}', json={}, headers=auth_headers["pm"])
 
-        assert is_valid(response, 201)
+        assert is_valid(response, 200)
```

Comment:
Should we be returning 200 here? I suppose depending on the perspective a 200 status is OK.

Reply:
Good catch. Since this is a failed update I think the status code should be:
422 Unprocessable Entity: The request was well-formed but was unable to be followed due to semantic errors.
or possibly 
400 Bad Request:  The server could not understand the request due to invalid syntax.

Reply:
Yeah, I think sending an empty payload to a patch request should automatically be rejected as a 400 bad request. Because at this point the entity in question could very well be processable, but there was no data to process. Hence, a bad request.